platform/spire/src/command.py

```diff
@@ -27,6 +27,13 @@ def fail(message: str, hint: str = None) -> None:
     raise CommandFailedException(message, hint)
 
 
+class MultipleExceptions(CommandFailedException):
+    def __init__(self, message, errs):
+        messages = [message] + [str(e) for e in errs]
+        hints = (e.hint for e in errs if hasattr(e, 'hint'))
+        super().__init__('\n'.join(messages), '\n'.join(hints))
```

Comment:
Is this guaranteed to be sensical? i.e. if the output message ends up having multiple lines and it goes:
```
[message]
[message for error 1]
[message for error 2]
[hint for error 1]
[hint for error 2]
```
It seems like it might be confusing which hint goes with which error.

Reply:
good question, will fix